workflow-ui/core-android/src/main/java/com/squareup/workflow1/ui/WorkflowRunnerViewModel.kt

```diff
@@ -65,23 +46,23 @@ internal class WorkflowRunnerViewModel<OutputT>(
       }
 
       val renderingsAndSnapshots = renderWorkflowIn(
-          config.workflow, scope, props,
-          initialSnapshot = snapshot,
-          interceptors = config.interceptors
+        config.workflow, scope, props,
```

Comment:
nit: while we're here, can you please name these parameters?

Reply:
> Can you please post an example of `#renderWorkflowInAndroid` in use? I'm having trouble figuring out what I'd do with the `scope` parameter.

I've already deleted it. Thought I had a use case where I couldn't use `ViewModel`, but I don't.

Reply:
Not much point, because the goal is to deprecate this entire mess. Or do you mean the params to `renderWorkflowIn`? Suggestions?

Reply:
Oh, I just mean that some variables have their argument names next to them (e.g. `initialSnapshot = snapshot`) and the first three parameters don't.